src/dsp_tools/commands/project/models/resourceclass.py

```diff
@@ -486,11 +486,14 @@ def getProperty(self, property_id: str) -> Optional[HasProperty]:
 
     def addProperty(
         self,
-        last_modification_date: DateTimeStamp,
+        # last_modification_date: DateTimeStamp,
         property_id: str,
         cardinality: Cardinality,
         gui_order: Optional[int] = None,
     ) -> DateTimeStamp:
+        encoded_res_iri = quote_plus(self.iri)
+        get_mod_date = self._con.get(f"{self.ROUTE}/{encoded_res_iri}")
+        latest_date = DateTimeStamp(get_mod_date["knora-api:lastModificationDate"]["@value"])
```

Comment:
I assumed that the last_modification_date was outdated. So I requested it again at this stage. However, in the debug mode, I saw that the dates were identical. So I don't understand the problem... Any ideas?